tools/run_tests/run_tests.py

```diff
@@ -483,24 +483,24 @@ def _compiler_options(self, use_docker, compiler):
             return ('jessie', [])
         elif compiler == 'gcc5.3':
             return ('ubuntu1604', [])
-        elif compiler == 'gcc7.4':
```

Comment:
when you are removing this, it seems that you should remove the cxx_ubuntu1804 image as well?

Btw, is there any particular reason for removing the gcc7.4 test?

Reply:
Removing cxx_ubuntu1804 makes sense. I'll do this. I'll update the description.